### PR TITLE
upgrade to broccoli-persistent-filter

### PR DESCRIPTION
**Pull request by stefanpenner**
- faster rebuilds
- lower cost for initial builds

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "MIT"
   ],
   "dependencies": {
-    "broccoli-filter": "^0.1.6",
+    "broccoli-persistent-filter": "^1.1.1",
```
Comment (toranb) [inline package.json]: should we make this 1.1.3 given the recent issues w/ 1.1.2 ?

**Comment by toranb** [comment]
Also -for future reference you are now owner :)

npm owner add stefanpenner broccoli-ember-hbs-template-compiler

**Comment by toranb** [comment]
Go ahead and merge /publish this when you feel it's ready dude

File: package.json (hunk 1 above)
Comment (stefanpenner) [inline package.json]: good catch.

**Comment by stefanpenner** [comment]
> Also -for future reference you are now owner :)

awesome

> Go ahead and merge /publish this when you feel it's ready dude

I feel pretty good about this one, but would love your +1 :)

**Comment by rwjblue** [comment]
@stefanpenner - LGTM

**Comment by toranb** [comment]
+1 dudes - get it merged/published/etc

**Comment by toranb** [comment]
@rwjblue it looks like your an owner on this also :)

**Comment by rwjblue** [comment]
Ya, I'm computer-less at the moment :)

**Pull request merged**